benchmarks/tracer/scenario.py

```diff
@@ -0,0 +1,43 @@
+import pyperf
+
+from ddtrace import tracer
+from ddtrace.filters import TraceFilter
+
+
+VARIANTS = [{}]
+
+
+def time_trace(loops, variant):
+    range_it = range(loops)
+    t0 = pyperf.perf_counter()
+    for _ in range_it:
+        with tracer.trace("1"):
+            with tracer.trace("2"):
+                with tracer.trace("3"):
+                    with tracer.trace("4"):
+                        with tracer.trace("5"):
+                            pass
+                        with tracer.trace("6"):
+                            pass
+                    with tracer.trace("7"):
+                        pass
+                with tracer.trace("8"):
+                    pass
+            with tracer.trace("9"):
+                pass
```

Comment:
Is it possible to generate this structure recursively? cc: @Kyle-Verhoog

Reply:
We considered recursion but were worried about it adding additional overhead to the measurement.